### ISSUE TITLE
Possible issue using 1Password replacement list without separators between words

### ISSUE THREAD

**Issue by sts10** [tmthrgd/passit#1]
Hello! Really neat to see my 1Password replacement wordlist getting used here. 

However, there could be a small/rare security concern: If users can use the 1Password replacement list to generate a passphrase _without_ any separator between the words (e.g. `emberbonfirerefuelingbrought`), there is a possibility that you get what I call a collision. As an example, let's say you ask for 4 words but you get "saucinessclericaloverreach". Here, "over" and "reach" are individual words on the list. But if "overreach" is also on the list, an attacker guessing 3-word combinations would guess our 4-word passphrase!

The EFF word lists get around this issue by [not having any prefix words on their lists](https://www.eff.org/deeplinks/2016/07/new-wordlists-random-passphrases), i.e. no word is an exact prefix of any other word. If a list doesn't have any prefix words in it, it is **uniquely decodable**, which solves the problem explained above ("over" would be removed, since it's a prefix of "overreach" -- [read more here](https://github.com/ulif/diceware#prefix-code)). 

I made my proposed 1Password list NOT uniquely decodable because [1Password's passphrase generator](https://1password.com/password-generator/) forces users to put punctuation between words, which solves the issue ("over-reach" can't be confused with the single word "overreach"). 

It's my understanding from the docs of passit that it's possible to use the STS10 wordlist without separator ("") between words. Is that correct? If so, you might want to consider removing the list. (If not, feel free to ignore this entire issue!)

Luckily, I have since created a series of wordlists that _are_ uniquely decodable that I call the [Orchard Street Wordlists](https://github.com/sts10/orchard-street-wordlists). [The Orchard Street Long List](https://github.com/sts10/orchard-street-wordlists/blob/main/lists/orchard-street-long.txt), which has 17,576 words, is closest to the STS10 list you currently use (18,208 words).

Let me know if you have any questions!

**Comment by sts10** [comment]
>If users can use the 1Password replacement list to generate a passphrase without any separator between the words (

Ah, so this does seem possible:
```bash
$ passphrase -s ""
canondivessmackedeyelashquidinventor
```

**Comment by tmthrgd** [comment]
Hi @sts10,

Thank you for raising this issue. As you discovered, it certainly is possible to use that wordlist without separators both using the commands and the library itself.

Thank you for all your work on these wordlists. I've gone and included the current version of the relevant Orchard Street wordlists in this project. 🙏🏻 

For backwards compatibility reasons I can't remove the 1password-replacement.txt wordlist. I gave it a lot of thought though and have decided to deprecate that particular wordlist and remove it from the CLI tools. The default for those tools will now be your Orchard Street Long List.

I wrestled with erroring only for that wordlist and when the risk is there but it was a pretty horrible conditional and unfriendly user experience.

Thank you again for raising this issue and all your work on password wordlists.
